listView.php

```diff
@@ -65,6 +65,11 @@
     Edit List Title
     </button>
 
+    <!-- button to edit title of an existing list -->
```

Comment:
For some reason, the board page is displaying two 'Add new task' buttons - one which links to the modal, and another which links to the full page.

Looks like you forgot to remove the older code - should be a quick and straightforward fix.

Reply:
Also, the comment is misleading - mentions 'button to edit list' when it's to add a new task.

Reply:
Will make changes - thanks.